magento2/.docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM supercid/magento-base:2.3.4
+
+ADD scripts/init-and-run.sh /usr/local/bin/init-and-run
+ADD scripts/xdebug.sh /usr/local/bin/xdebug
+
+RUN echo 'www-data ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
```

Comment:
Shouldn't this be in the Apache image or the base image?

Reply:
Moved there